core/templates/pages/creator-dashboard-page/modal-templates/upload-activity-modal.controller.spec.ts

```diff
@@ -53,8 +53,9 @@ describe('Upload Activity Modal Controller', function() {
       size: 100,
       name: 'file.mp3'
     };
-    // The document has strict type rules which will fail typescript lint tests
-    // without the @ts-ignore.
+    // There is a typescript error here because the actual getElementById
+    // returns more properties than just files. We need to suppress this error
+    // because we need only files for testing.
```

Comment:
... need only the "files" property for testing.

That said -- please file an issue and add a TODO here for changing the code being tested to not use DOM methods. This seems to be an anti-pattern in Angular and is the sort of thing the TS compiler checks would help with avoiding in the future (at the very least, a reviewer would hopefully see this and ask questions).

Reply:
done. Filed #10113.